src/app/shared/data-input/datetime/datetime-data-input.component.ts

```diff
@@ -93,7 +93,11 @@ export class DatetimeDataInputComponent implements OnChanges {
     if (changes.value && String(this.value).length === 1) {
       // show value entered into hidden input without any changes
       const input = this.dateInput;
-      setTimeout(() => (input.nativeElement.value = this.value));
+      setTimeout(() => {
+        if (input) {
+          input.nativeElement.value = this.value;
```

Comment:
For the future, there exists shortcut: `(input && (input.nativeElement.value = this.value)))`.

Reply:
That's what I wanted to use!